README.md

```diff
@@ -10,7 +10,7 @@ Installs and configures Jenkins CI master & node slaves. Resource providers to s
 
 - Debian 7+ (Package installs require 9+ due to dependencies)
 - Ubuntu 14.04+ (Package installs require 16.04+ due to dependencies)
-- RHEL/CentOS/Scientific/Oracle 6+
+- RHEL/CentOS/SUSE/Scientific/Oracle 6+
```

Comment:
SUSE should go on its own line since the 6+ part doesn't apply to SUSE

Reply:
Also you should clarify what SUSE we're talking about here. SLES or openSUSE

Reply:
Whoops! This was totally my bad, I'll push this up.